css/typography.css

```diff
@@ -85,8 +91,8 @@ h5, .h5 {
     line-height: 1.3;
 }
 
-h2, h3, h4, h5, h6,
-.h2, .h3, .h4, .h5, .h6 {
+h4, h5, h6,
+.h4, .h5, .h6 {
```

Comment:
Why remove `h2` from here? Seems like this block should just be moved above the specific styling so it acts as a default.